front/src/applications/operationalStudies/v2/components/ScenarioHeader.tsx

```diff
@@ -12,18 +12,30 @@ import { useModal } from 'common/BootstrapSNCF/ModalSNCF';
 import AddAndEditScenarioModal from 'modules/scenario/components/AddOrEditScenarioModal';
 import useAuth from 'utils/hooks/useAuth';
 
+type Board = 'trains' | 'map' | 'macro' | 'std' | 'sdd' | 'table' | 'conflicts';
+
 type ScenarioHeaderProps = {
   scenario: ScenarioResponse;
   infra: InfraWithState;
+  toggleTimetable: () => void;
+  toggleConflictsList: () => void;
```

Comment:
The state of each board is duplicated in `Scenario` and in `ScenarioHeader`. `Scenario` has one boolean per board, and `ScenarioHeader` has a list of active boards.

We should probably lift up the state here?

```ts
type ScenarioHeaderProps = {
  …
  activeBoards: Board[];
  toggleBoard: (Board) => void;
};
```

I'm fine keeping it as-is for now, if we're going to fix it later. (Maybe leave a TODO?)